modules/db/src/main/scala/gem/dao/ObservationDao.scala

```diff
@@ -21,6 +21,7 @@ object ObservationDao {
                     ${o.id.index},
                     ${o.title},
                     ${o.instrument})
+       ON CONFLICT DO NOTHING
```

Comment:
Yeah I don't think this is what we want. This should be an error. You can delete from the program table and it will cascade everything.